<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pqClientMainWindow</class>
 <widget class="QMainWindow" name="pqClientMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1152</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../Qt/Components/Resources/pqComponents.qrc">
    <normaloff>:/pqWidgets/Icons/pqAppIcon64.png</normaloff>:/pqWidgets/Icons/pqAppIcon64.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="margin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="pqTabbedMultiViewWidget" name="MultiViewWidget" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1152</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuSources">
    <property name="title">
     <string>&amp;Sources</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuFilters">
    <property name="title">
     <string>&amp;Filters</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_View">
    <property name="title">
     <string>&amp;View</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>&amp;Tools</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_Macros">
    <property name="title">
     <string>&amp;Macros</string>
    </property>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Edit"/>
   <addaction name="menu_View"/>
   <addaction name="menuSources"/>
   <addaction name="menuFilters"/>
   <addaction name="menuTools"/>
   <addaction name="menu_Macros"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="pqStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="pipelineBrowserDock">
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Pipeline Browser</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="pqPipelineBrowserWidget" name="pipelineBrowser">
    <property name="contextMenuPolicy">
     <enum>Qt::ActionsContextMenu</enum>
    </property>
   </widget>
  </widget>
  <widget class="QDockWidget" name="statisticsDock">
   <property name="allowedAreas">
    <set>Qt::BottomDockWidgetArea|Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Statistics Inspector</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="pqDataInformationWidget" name="statisticsView"/>
  </widget>
  <widget class="QDockWidget" name="animationViewDock">
   <property name="windowTitle">
    <string>Animation View</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="pqPVAnimationWidget" name="animationView"/>
  </widget>
  <widget class="QDockWidget" name="comparativePanelDock">
   <property name="windowTitle">
    <string>Comparative View Inspector</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="pqComparativeVisPanel" name="comparativeVisPanel"/>
  </widget>
  <widget class="QDockWidget" name="collaborationPanelDock">
   <property name="windowTitle">
    <string>Collaboration Panel</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="pqCollaborationPanel" name="collaborationPanel"/>
  </widget>
  <widget class="QDockWidget" name="objectInspectorDock">
   <property name="windowTitle">
    <string>Properties</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="pqObjectInspectorWidget" name="objectInspector"/>
  </widget>
  <widget class="QDockWidget" name="displayDock">
   <property name="windowTitle">
    <string>Display</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="displayWidgetFrame">
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QScrollArea" name="displayScrollArea">
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="pqDisplayProxyEditorWidget" name="displayWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>77</width>
          <height>152</height>
         </rect>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="informationDock">
   <property name="windowTitle">
    <string>Information</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="informationWidgetFrame">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QScrollArea" name="informationScrollArea">
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="pqProxyInformationWidget" name="informationWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>77</width>
          <height>152</height>
         </rect>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="memoryInspectorDock">
   <property name="windowTitle">
    <string>Memory Inspector</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="pqMemoryInspectorPanel" name="dockWidgetContents"/>
  </widget>
  <widget class="QDockWidget" name="propertiesDock">
   <property name="windowTitle">
    <string>Properties</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="pqPropertiesPanel" name="propertiesPanel"/>
  </widget>
  <widget class="QDockWidget" name="multiBlockInspectorDock">
   <property name="windowTitle">
    <string>Multi-block Inspector</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="pqMultiBlockInspectorPanel" name="multiBlockInspectorPanel"/>
  </widget>
  <widget class="QDockWidget" name="colorMapEditorDock">
   <property name="windowTitle">
    <string>Color Map Editor</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="pqColorMapEditor" name="colorMapEditorPanel"/>
  </widget>
  <widget class="QDockWidget" name="selectionDisplayDock">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="windowTitle">
    <string>Selection Display Inspector</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="pqFindDataSelectionDisplayFrame" name="selectionDisplayWidget">
    <property name="useVerticalLayout" stdset="0">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>pqPipelineBrowserWidget</class>
   <extends>QWidget</extends>
   <header>pqPipelineBrowserWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>pqTabbedMultiViewWidget</class>
   <extends>QWidget</extends>
   <header>pqTabbedMultiViewWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>pqPVAnimationWidget</class>
   <extends>QWidget</extends>
   <header>pqPVAnimationWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>pqDataInformationWidget</class>
   <extends>QWidget</extends>
   <header>pqDataInformationWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>pqStatusBar</class>
   <extends>QStatusBar</extends>
   <header>pqStatusBar.h</header>
  </customwidget>
  <customwidget>
   <class>pqComparativeVisPanel</class>
   <extends>QWidget</extends>
   <header>pqComparativeVisPanel.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>pqCollaborationPanel</class>
   <extends>QWidget</extends>
   <header>pqCollaborationPanel.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>pqObjectInspectorWidget</class>
   <extends>QWidget</extends>
   <header>pqObjectInspectorWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>pqDisplayProxyEditorWidget</class>
   <extends>QWidget</extends>
   <header>pqDisplayProxyEditorWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>pqProxyInformationWidget</class>
   <extends>QWidget</extends>
   <header>pqProxyInformationWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>pqMemoryInspectorPanel</class>
   <extends>QWidget</extends>
   <header>pqMemoryInspectorPanel.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>pqPropertiesPanel</class>
   <extends>QWidget</extends>
   <header>pqPropertiesPanel.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>pqMultiBlockInspectorPanel</class>
   <extends>QWidget</extends>
   <header>pqMultiBlockInspectorPanel.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>pqColorMapEditor</class>
   <extends>QWidget</extends>
   <header>pqColorMapEditor.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>pqFindDataSelectionDisplayFrame</class>
   <extends>QWidget</extends>
   <header>pqFindDataSelectionDisplayFrame.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../Qt/Components/Resources/pqComponents.qrc"/>
 </resources>
 <connections/>
</ui>
